src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3LinuxDriver.cs

```diff
@@ -654,7 +655,7 @@ private void Initialize()
             }
 
             // try and open /dev/gpiomem
-            fileDescriptor = Interop.open(GpioMemoryFilePath, FileOpenFlags.O_RDWR | FileOpenFlags.O_SYNC);
+            fileDescriptor = global::Interop.open(GpioMemoryFilePath, FileOpenFlags.O_RDWR | FileOpenFlags.O_SYNC);

```

Comment:
Can you help me understand why we now need to use the global namespace here? What is clashing with this?

Reply:
So this was necessary after moving for example the class `SafeChipHandle` to it's file location namespace, so from `namespace System.Device.Gpio;` to `namespace System.Device.Gpio.Interop.Unix;` (I did this be consistent but since you do not have the convention to have the namespace same as file location I may just revert this)
I have not found yet why after that the "global" was needed...